validation/src/main/java/org/craftercms/commons/validation/annotations/param/EsapiValidationType.java

```diff
@@ -24,7 +24,9 @@ public enum EsapiValidationType {
     HTTPParameterName("HTTPParameterName"),
     HTTPURI("HTTPURI"),
     SITE_ID("SITEID"),
-    USERNAME("USERNAME");
+    EMAIL("EMAIL"),
```

Comment:
@phuongnq I forgot to mention, please also add error messages here:

https://github.com/craftercms/commons/blob/develop/validation/src/main/resources/crafter/commons/validation/errors.properties#L44